apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -0,0 +1,779 @@
+                                    data-placement="auto"
+                                    title="Fingerprint ${row.client_fingerprint_number}"
+                                    data-content="
+                                        <div>
+                                            IP Address: ${row.client_fingerprint?.ip_address}
+                                        </div>
+                                        <div>
+                                            Session ID: ${row.client_fingerprint?.user_session_id}
+                                        </div>
+                                        <div>
+                                            User Agent: ${row.client_fingerprint?.user_agent}
+                                        </div>
+                                        "
+                                    data-trigger="focus"
+                                  >
+                                    ${row.client_fingerprint_number}
+                                  </a>
+                                </td>
+                              `
+                            : html`<td>&mdash;</td>`}`
+                        : ''}
+                      <td><span class="badge color-${row.event_color}">${row.event_name}</span></td>
+                      <td>
+                        ${row.qid
+                          ? html`
+                              <a href="${resLocals.urlPrefix}/question/${row.question_id}/">
+                                I-${row.instructor_question_number} (${row.qid})
+                              </a>
+                            `
+                          : ''}
+                      </td>
+                      <td>
+                        ${row.student_question_number
+                          ? row.variant_id
+                            ? html`
+                                <a
+                                  href="${resLocals.plainUrlPrefix}/course_instance/${resLocals
+                                    .course_instance
+                                    .id}/instance_question/${row.instance_question_id}/?variant_id=${row.variant_id}"
+                                >
+                                  S-${row.student_question_number}#${row.variant_number}
+                                </a>
+                              `
+                            : html`S-${row.student_question_number}}`
+                          : ''}
+                      </td>
+                      ${row.event_name !== 'External grading results'
+                        ? html`<td style="word-break: break-all;">
+                            ${row.data != null ? JSON.stringify(row.data) : ''}
+                          </td>`
+                        : html`
+                            <td>
+                              <a
+                                class="btn btn-primary"
+                                href="<%= urlPrefix %>/grading_job/<%= row.data.id %>"
```

Comment:
You missed some EJS syntax here.

Reply:
Removed usage of EJS in [4f716c1](https://github.com/PrairieLearn/PrairieLearn/pull/9277/commits/4f716c16f38f011939f082816aa9c5c086f0460a).